platform/landing-page/src/main/java/org/codice/ddf/landing/LandingPage.java

```diff
@@ -130,6 +135,18 @@ public void setExternalUrl(String externalUrl) {
     this.externalUrl = externalUrl;
   }
 
+  public void setSourceKeyword(String sourceKeyword) {
+    this.sourceKeyword = sourceKeyword;
+  }
+
+  public void setkeywordRegistry(KeywordRegistry keywordRegistry) {
```

Comment:
✏️ `setKeywordRegistry`

Reply:
❗️ This is actually a necessary change for the blueprint I believe 
✏️ include the `@SuppressWarnings` for "unused" method

Reply:
It definitely is a necessary change